module/scripts/rdf-schema-alignment.js

```diff
@@ -1,336 +1,363 @@
+        self._renderBody(body);
+        DialogSystem.dismissUntil(self._level);
+      } else {
+        return;
+      }
+    }, function(rdfSkeleton) {
+      $.getJSON("command/core/get-operations?" + $.param({
+        project: theProject.id
+      }), function(json) {
+        var operations_old = json.entries.map(function(val) {
+          return val['operation'];
+        });
         
-        Refine.postProcess(
-                "rdf-extension",
-                "save-rdf-schema",
-                {},
-                { schema: JSON.stringify(schema) },
-                {},
-                {   
-                        onDone: function() {
-                        theProject.overlayModels.rdfSchema = schema;
-                        }
-                }
-        );
-   });
-    
-    elmts._share_skeleton.click(function(e){
-    	e.preventDefault();
-    	var schema = self.getJSON();
-    	rdfSkeletonMetadataDialog = new RdfSkeletonMetadataDialog(function(){
-            var schema = self.getJSON();
-            var title = $("input#title").val();
-            var fileType = $("input#fileType").val();
-            
-            if (fileType != null){
-                $.post("command/rdf-extension/save-rdf-skeleton",
-                        {model: JSON.stringify(self.getJSON()), 
-                    projectId: theProject.id, fileType: fileType, title: title},
-                    function(data){
-                        alert('RDF skeleton saved sucessfully');
-                    }
-                );
+        if (operations_old.length == 1) {
+          operations_old = [];
+        } else {
+          operations_old = operations_old.slice(-5);
+        }
+        var operations_new = JSON.parse(rdfSkeleton['skeleton']).operations;
+        Refine.postCoreProcess("apply-operations", {}, {
+          operations: JSON.stringify(operations_old.concat(operations_new))
+        }, {
+          everythingChanged: true
+        }, {
+          onDone: function(o) {
+            if (o.code == "pending") {
+              Refine.update({
+                everythingChanged: true
+              });
             }
-            
-            
-            Refine.postProcess(
-                    "rdf-extension",
-                    "save-rdf-schema",
-                    {},
-                    { schema: JSON.stringify(schema) },
-                    {},
-                    {   
-                        onDone: function() {
-                           theProject.overlayModels.rdfSchema = schema;
-                        }
-                    }
-                );
-    	});
-    });
+          }
+        });
+      });
+      var prefixes = [];
+      var new_prefixes = JSON.parse(rdfSkeleton['skeleton']).prefixes;
+      if (self._schema.prefixes != undefined) {
+        prefixes = prefixes.concat(self._schema.prefixes).concat(new_prefixes);
+      } else {
+        prefixes = prefixes.concat(new_prefixes);
+      }
```

Comment:
changes are in line 125 - 131. first i concatenate the prefixes then i put the unique values in another array . This is the array returned to the front end